infra/write_env.ps1

```diff
@@ -16,14 +16,35 @@ Add-Content -Path $ENV_FILE_PATH -Value "AZURE_COSMOSDB_OAUTH_CONTAINER=$(azd en
 Add-Content -Path $ENV_FILE_PATH -Value "APPLICATIONINSIGHTS_CONNECTION_STRING=$(azd env get-value APPLICATIONINSIGHTS_CONNECTION_STRING)"
 Add-Content -Path $ENV_FILE_PATH -Value "MCP_AUTH_PROVIDER=$(azd env get-value MCP_AUTH_PROVIDER)"
 $KEYCLOAK_REALM_URL = azd env get-value KEYCLOAK_REALM_URL 2>$null
+# Filter out ERROR messages from azd output (azd writes errors to stdout, not stderr)
+if ($KEYCLOAK_REALM_URL -and $KEYCLOAK_REALM_URL.Contains("ERROR:")) {
```

Comment:
Can we add a wrapper for this, since it's done twice?

Reply:
turned into wrapper in latest commit